pom.xml

```diff
@@ -230,5 +231,117 @@
         </plugins>
       </build>
     </profile>
+
+    <!-- Thorntail Profile -->
+    <profile>
+    <id>thorntail</id>
+      <properties>
+        <version.thorntail>2.0.0.Final</version.thorntail>
+        <version.docker-maven-plugin>0.26.1</version.docker-maven-plugin>
+
+
+      </properties>
+
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>io.thorntail</groupId>
+            <artifactId>bom-all</artifactId>
+            <version>${version.thorntail}</version>
+            <scope>import</scope>
+            <type>pom</type>
+          </dependency>
+        </dependencies>
+      </dependencyManagement>
+
+      <dependencies>
+        <!-- Thorntail Fractions for the implementation -->
+        <dependency>
+          <groupId>io.thorntail</groupId>
+          <artifactId>microprofile</artifactId>
+          <!-- that should not be needed.See THORN-2015 -->
+          <exclusions>
+            <exclusion>
+              <groupId>org.wildfly.swarm</groupId>
+              <artifactId>config-api-runtime</artifactId>
+            </exclusion>
+          </exclusions>
+        </dependency>
+<!-- the next ones should be in the above, but unfortunately are missing in TT 2.0
+    See THORN-2093
+-->
+        <dependency>
+             <groupId>io.thorntail</groupId>
```

Comment:
Is this MP Open API jar? The dependency on MP 1.3 should do it.

Reply:
I hope this whole "additional things" section will go away soon :-)